src/API/scenarios/scenario.test.js

```diff
@@ -19,7 +19,15 @@ describe('## scenario APIs', () => {
             _id: mongoose.Types.ObjectId('000000000000000000000000'),
             name: 'Greeting script',
             description: 'will greet you in a brand new world',
-            body: 'console.log("Hello Scripto World!");'
+            body: 'console.log("Hello Scripto World!");',
+            active: true,
+            isConvertible: false,
+            jsCode: '',
+            generatedJson: null,
+            sourceType: '',
+            actions: [],
+            conditions: [],
+            logicalOperator: ''
```

Comment:
!fix
why do we need `generatedJson` aside with `actions`, `conditions`, `logicalOperator`
(not sure what `sourceType` holds)